src/contracts/core/DelegationManager.sol

```diff
@@ -714,9 +940,9 @@ contract DelegationManager is
             delegatedTo: operator,
             withdrawer: withdrawer,
             nonce: nonce,
-            startBlock: uint32(block.number),
+            startTimestamp: uint32(block.timestamp),
```

Comment:
queueTimestamp?

Reply:
kinda prefer as is, so we remember this field is the same one used as in M2